x-pack/plugins/uptime/public/components/overview/kuery_bar/kuery_bar.tsx

```diff
@@ -105,16 +105,7 @@ export function KueryBar({
           query: inputValue,
           selectionStart: selectionStart || 0,
           selectionEnd: selectionStart || 0,
-          boolFilter: [
-            {
-              range: {
-                '@timestamp': {
-                  gte: dateRangeStart,
-                  lte: dateRangeEnd,
-                },
-              },
-            },
-          ],
+          useTimeRange: true,
```

Comment:
What time range it will use, how will it know what date range is being selected in uptime? 

i suspect you are planing on using shared service?

Reply:
I saw that you were already using the shared service in `update_date_picker`. Is it not the case?

Reply:
yes that's indeed true, my point is that what if some solution isn't doing that? they will have to pass those params alongside, do we have a fall back approach?

Reply:
See the comment I just posted for APM https://github.com/elastic/kibana/pull/81515#discussion_r517934051 as I suspect they don't use the global timepicker.

They would have to pass in `useTimeFilter: false` explicitly. But as far as I can see, there's only one place that would have to do that (APM)

Reply:
@shahzad31 let me know if you think this solution is ok or if I should revert it :)